src/main/java/com/weirddev/testme/intellij/template/context/Method.java

```diff
@@ -156,5 +156,12 @@ public boolean hasReturn(){
     public boolean hasParams() {
         return !methodParams.isEmpty();
     }
+
+    /**
+     * get method exception types
+     */
+    public String getMethodExceptionTypes() {
```

Comment:
this getter should be generated by Lombok. as all getters for fields in this class.
In case the getter is not visible during development - I warmly recommend installing the [Lombok plugin](https://plugins.jetbrains.com/plugin/6317-lombok)